master/docs/manual/cfg-www.rst

```diff
@@ -268,6 +268,8 @@ The default templates are very much configurable via the following options.
     {
         "left_text": "Build Status",  # text on the left part of the image
         "left_color": "#555",  # color of the left part of the image
+        "left_size": "40", # override autodetected font width, omit if not required
```

Comment:
could you please elaborate a bit more how this change improves things? compared to the current state.